### PR TITLE
feat: functionality to remove added educations from the onboarding screen

### PR DESCRIPTION
**Pull request by sourabhrathourr**
### Issue: 
[LET-10: feat: Functionality to remove added educations from the onboarding screen](https://linear.app/letraz/issue/LET-10/feat-functionality-to-remove-added-educations-from-the-onboarding)

### Description:
The current implementation of the onboarding page's education screen allows users to add multiple education histories but lacks the ability to remove previously added entries. This feature is necessary to enable users to correct or adjust their education information as needed.

### Changes Made:
- Added a "Remove" icon next to each added education entry.
- Implemented functionality to remove the corresponding education entry from the screen and user's data upon clicking the "Remove" icon.
- Ensured smooth UI updates to reflect the removal of the entry.
- Implemented proper data handling and error prevention mechanisms for the removal process.
- Tested the feature across all supported browsers and devices to ensure consistent functionality.

### Closing Note:
The functionality to remove added educations from the onboarding screen has been successfully implemented. Users can now easily correct or adjust their education information by removing previously added entries, ensuring accurate data collection during the onboarding process.

### CODE REVIEW

**Review by sahilverma-dev — APPROVED** [review]
Working fine

File: components/onboarding/Education.tsx [hunk 1]
```diff
@@ -8,9 +8,14 @@ import {motion} from 'framer-motion'
 import {useAutoAnimate} from '@formkit/auto-animate/react'
 import {educations} from '@/db/schema'
 import {months} from '@/constants'
+import {X} from 'lucide-react'
+import PopConfirm from '../ui/pop-confirm'
```
Comment (pingSubhajit) [inline components/onboarding/Education.tsx:12]: Try importing the component using import alias instead of relative path

**Review by pingSubhajit — CHANGES_REQUESTED** [review]
Everything else looks fine. Just make the necessary changes and it will be good to merge.

File: components/onboarding/Education.tsx (hunk 1 above)
Comment (sourabhrathourr) [reply components/onboarding/Education.tsx]: Done 👍🏻

**Review by pingSubhajit — APPROVED** [review]
This looks fine now. I recommend merging.

**Pull request merged**